tests/test_convert.py

```diff
@@ -0,0 +1,23 @@
+import sys
```

Comment:
copy a standard header from the other unit tests, like copyright and docstr

Reply:
copied.